src/main/java/com/parsing/model/LotResult.java

```diff
@@ -34,19 +27,42 @@ public class LotResult {
     @Column(name = "id")
     @GeneratedValue(generator = "UUID")
     private UUID id;
+
+    @ManyToOne
+    private Participant buyer;
+
+    @ManyToOne
+    private Participant seller;
+
+    @Enumerated(value = EnumType.STRING)
+    @Column(name = "status")
+    private LotStatus lotStatus;
+
     @Column(name = "dk")
     private String dk;
-    @Column(name = "url")
-    private String url;
-    @Column(name = "pdf_link")
-    private String pdfLink;
+
+    @Column(name = "lot_total_price")
+    private BigDecimal lotTotalPrice;
+
+    @ManyToMany
+    private List<Participant> participants;
+
+    @Column(name = "lot_url")
+    private String lotURL;
+
+    @Column(name = "pdf_url")
+    private String pdfURL;
+
+    @OneToMany(mappedBy = "lotInfo", cascade = CascadeType.ALL)
```

Comment:
I'm not sure you have to add in LotInfo information regarding LotInfoItem. LotInfoItem only for LotInfo.

Reply:
Done.